src/clients/inst_autoimage.rb

```diff
@@ -1,73 +0,0 @@
-# File:  clients/inst_autoimage.ycp
-# Package:  Auto-installation
-# Author:      Anas Nashif <nashif@suse.de>
-# Summary:  Imaging
-#
-# $Id$
-module Yast
-  class InstAutoimageClient < Client
-    def main
-      textdomain "autoinst"
-
-      Yast.import "Installation"
-      Yast.import "Progress"
-      Yast.import "Report"
-      Yast.import "AutoinstImage"
-      Yast.import "AutoinstSoftware"
-      Yast.import "AutoinstScripts"
-      Yast.import "AutoinstConfig"
-
-      Yast.include self, "autoinstall/io.rb"
-
-      AutoinstScripts.Write("postpartitioning-scripts", false)
```

Comment:
ah, no, This change is bad. This basically kills post partitioning scripts. I am fine with removing autoimage, but you have to move this call otherwise postpartitioning scripts won't be executed at all.